src/components/HomePage/HomePage.css

```diff
@@ -39,6 +39,20 @@
   position: relative;
 }
 
+.HomePage .empty-projects {
+  display: flex;
+  width: 100%;
+  height: 200px;
+  border-radius: 12px;
+  border: 1px solid var(--bluish-steel);
+  align-items: center;
+  justify-content: center;
+  color: var(--bluish-steel);
```

Comment:
```suggestion
  color: var(--secondary-text);
```

Reply:
let's use `secondary-text`, which is the same color (for this theme) but is more semantically correct